service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/InMemoryPolicyEvaluationCacheTest.java

```diff
@@ -73,9 +70,7 @@ public void testSetPolicyPolicySetChangedTimestamp() throws Exception {
         this.cache.set(key, value);
 
         Map<String, String> evalCache = (Map<String, String>) getInternalState(this.cache, "evalCache");
-        assertEquals(evalCache.size(), 0);
-        Map<String, String> timestampCache = (Map<String, String>) getInternalState(this.cache, "timestampCache");
-        assertEquals(timestampCache.size(), 1);
+        assertEquals(evalCache.size(), 1);
```

Comment:
Current implementation of the in-memory cache stores eval result and timestamps in the same maps, not in different maps as before.  There is no need to have different tests for setting eval result and setting timestamps.